packages/ag-charts-community/src/chart/series/cartesian/barSeries.ts

```diff
@@ -654,6 +655,7 @@ export class BarSeries extends AbstractBarSeries<
             lineDash: highlightStyle?.lineDash ?? properties.lineDash ?? [],
             lineDashOffset: highlightStyle?.lineDashOffset ?? properties.lineDashOffset,
             cornerRadius,
+            defaultColorRange: properties.defaultColorRange,
```

Comment:
What does this do?

Reply:
It's used when colorStops aren't configured